public/js/index.js

```diff
@@ -3,3 +3,82 @@
  *= require jquery/dist/jquery.js
  *= require bootstrap/dist/js/bootstrap.js
  */
+
+ var lock = new Auth0Lock('pZJp3rVJ8wRvVLrX30HEOOHZ3HSEASec', 'hexlet-interviews.eu.auth0.com');
```

Comment:
I think we will remove auth with third-party service before next interview, and now our app turned off